tests/details/details-page.spec.ts

```diff
@@ -70,20 +70,82 @@ test(`Load theater: image (single)`, async ({ detailsPage }) => {
   await detailsPage.imageCarouselMultipleImageDisplay(false);
 });
 
-test(`Load theater: radio`, async ({ detailsPage }) => {
-  await detailsPage.gotoPage('WGBH_89_7_FM_20210918_040000');
-  await detailsPage.radioPlayerTheaterDisplay();
+test(`Load theater: radio as priv'd user`, async ({
+  loginPage,
+  detailsPage,
+}) => {
+  await test.step(`Do login as priv'd user`, async () => {
+    await loginPage.loginAs('privs');
+  });
+  await test.step(`Go to radio details page and verify priv'd user borrow program`, async () => {
+    await detailsPage.gotoPage('WGBH_89_7_FM_20210918_040000');
+    await detailsPage.radioPlayerTheaterDisplay();
+    await detailsPage.verifyRadioBorrowProgramAvailable();
+  });
 });
 
-test(`Load theater: tv`, async ({ detailsPage }) => {
-  test.info().annotations.push({
-    type: 'Test',
-    description: 'This is deferred for now',
+test(`Load theater: radio as regular patron user`, async ({
+  loginPage,
+  detailsPage,
+}) => {
+  await test.step(`Do login as regular patron user`, async () => {
+    await loginPage.loginAs('patron');
   });
+  await test.step(`Go to radio details page and verify priv'd user borrow program`, async () => {
+    await detailsPage.gotoPage('WGBH_89_7_FM_20210918_040000');
+    await detailsPage.radioPlayerTheaterDisplay();
+    await detailsPage.verifyRadioBorrowProgramUnavailable();
+  });
+});
 
-  await detailsPage.gotoPage(
-    'CSPAN3_20170413_154200_Discussion_Focuses_on_Sesame_Street_and_Autism',
-  );
+test(`Load theater: radio as guest/not logged in user`, async ({ detailsPage }) => {
+  await test.step(`Go to radio details page and verify priv'd user borrow program`, async () => {
+    await detailsPage.gotoPage('WGBH_89_7_FM_20210918_040000');
+    await detailsPage.radioPlayerTheaterDisplay();
+    await detailsPage.verifyRadioBorrowProgramUnavailable();
+  });
+});
+
+test(`Load theater: tv as priv'd user`, async ({
+  loginPage,
+  detailsPage,
+}) => {
+  await test.step(`Do login as priv'd user`, async () => {
+    await loginPage.loginAs('privs');
+  });
+  await test.step(`Go to tv details page and verify priv'd user borrow program`, async () => {
+    await detailsPage.gotoPage(
+      'CSPAN3_20170413_154200_Discussion_Focuses_on_Sesame_Street_and_Autism',
```

Comment:
Suggestion: As above, create and use constant for item identifier.

Reply:
we can have this in another ticket